src/status_im2/contexts/onboarding/create_password/style.cljs

```diff
@@ -1,14 +1,66 @@
 (ns status-im2.contexts.onboarding.create-password.style
-  (:require [quo2.foundations.colors :as colors]
-            [react-native.platform :as platform]))
+  (:require [quo2.foundations.colors :as colors]))
 
-(def navigation-bar {:height 56})
+(def image-background
+  {:height "100%"
+   :width  "100%"})
```

Comment:
Not sure about this one, but maybe just `{:flex 1}`

Reply:
I've tried it, but makes it doesn't look as the designs, because the flex can be shrinked.

This background will be replaced when https://github.com/status-im/status-mobile/issues/15274 it's done.